fastlane_core/spec/itunes_transporter_spec.rb

```diff
@@ -50,11 +50,17 @@ def shell_download_command(provider_short_name = nil, jwt: nil)
     end
 
     def shell_provider_id_command(jwt: nil)
+      # Ruby doesn't escape "+" with Shellwords.escape from 2.7 https://bugs.ruby-lang.org/issues/14429
+      escaped_password = if RUBY_VERSION > "2.7.0"
```

Comment:
2.7.0 included? Or in a subsequent patch?

Reply:
Ahhh very nice catch!!! I was testing with 2.7.2 so had no issue with this but yes this should include 2.7.0 🙇 

```suggestion
      escaped_password = if RUBY_VERSION >= "2.7.0"
```